megamek/resources/megamek/common/templates/tro/aero.ftl

```diff
@@ -14,6 +14,9 @@ ${formatBasicDataRow("Engine", engineName, engineMass)}
 	Safe Thrust: ${safeThrust}
 	Max Thrust: ${maxThrust}
 ${formatBasicDataRow("Structural Integrity:", si, "")}
+<#if isVSTOL && isConventional>
```

Comment:
`isVSTOL` is never defined in the template data model, causing the text version to fail. This also needs to exclude fixed wing support, which gain VSTOL capabilities through a chassis modification, which is listed elsewhere. That's probably best done by changing the definition of isConventional to exclude support vehicles.

Reply:
Okay, now I'm sure there's something up, cause I know I put in isVSTOL (mostly because I forgot it originally and got an error during testing). I think my last commit might not have gone through...